packages/app/src/domain/tested/reproduction-chart-tile.tsx

```diff
@@ -0,0 +1,66 @@
+import { useIntl } from '~/intl';
+import { colors } from '~/style/theme';
+import { NationalReproduction } from '@corona-dashboard/common';
+import { ChartTileWithTimeframe } from '~/components-styled/chart-tile';
+import { TimeSeriesChart } from '~/components-styled/time-series-chart';
+import { TimeframeOption } from '~/utils/timeframe';
+
+interface ReproductionChartTileProps {
+  data: NationalReproduction;
+  timeframeOptions?: TimeframeOption[];
+  timeframeInitialValue?: TimeframeOption;
+}
+
+export function ReproductionChartTile({
+  data: __data,
+  timeframeOptions = ['all', '5weeks'],
+  timeframeInitialValue = 'all',
+}: ReproductionChartTileProps) {
+  const { siteText } = useIntl();
+  const text = siteText.reproductiegetal;
+
+  /**
+   * There is no data for the last 2 weeks so we are getting a slice
+   * of all the values before the first datapoint with a null value to
+   * display in the chart
+   */
+  const values = __data.values.slice(
+    0,
+    __data.values.map((x) => x.index_average).indexOf(null)
+  );
```

Comment:
I think there is an easier way. How about this? (using isPresent from ts-is-present)

```suggestion
  const values = __data.values.filter((x) => isPresent(x.index_average))
  );
```

Reply:
Hmm I think I did it that way to avoid potentially removing values in the middle of the data set, but since this uses a linear scale it shouldn't matter. I'll use what you proposed 👍